lib/osf-components/addon/components/addons-service/manager/component.ts

```diff
@@ -33,11 +42,40 @@ export default class AddonsServiceManagerComponent extends Component<Args> {
     node = this.args.node;
     @tracked addonServiceNode?: InternalResourceModel;
 
-    @tracked addonProviders: Provider[] = [];
+    possibleFilterTypes = Object.values(FilterTypes);
+    @tracked filterText = '';
+    @tracked activeFilterType: FilterTypes = FilterTypes.STORAGE;
+    @tracked storageProviders: Provider[] = [];
+    @tracked allAddonProviders: Provider[] = [];
 
     @tracked pageMode?: PageMode;
     @tracked selectedProvider?: Provider;
 
+    @action
+    filterByAddonType(type: FilterTypes) {
+        this.activeFilterType = type;
+    }
+
+    get filteredAddonProviders() {
+        let possibleProviders: any[] = [];
+        switch (this.activeFilterType) {
+        case FilterTypes.STORAGE:
+            possibleProviders = this.storageProviders;
+            break;
+        case FilterTypes.CITATION_MANAGER:
+        case FilterTypes.CLOUD_COMPUTING:
+        case FilterTypes.DMP:
+        case FilterTypes.ELECTRONIC_LAB_NOTEBOOK:
+        case FilterTypes.PRESERVATION:
```

Comment:
I don't think we need these cases or FilterTypes. I know they're in the mockups, but we won't have any of these by launch, so there's no reason to have them in the filter. We'll add them once we have addons to match.